test.js

```diff
@@ -124,6 +124,47 @@ t.test('fastify-oauth2', t => {
     makeRequests(t, fastify)
   })
 
+  t.test('register multiple plugins', t => {
+    const fastify = createFastify({ logger: { level: 'silent' } })
+    const options = {
+      name: 'githubOAuth2',
+      credentials: {
+        client: {
+          id: 'my-client-id',
+          secret: 'my-secret'
+        },
+        auth: oauthPlugin.GITHUB_CONFIGURATION
+      },
+      startRedirectPath: '/login/github',
+      callbackUri: 'http://localhost:3000/callback',
+      scope: ['notifications']
+
+    }
+
+    fastify.register(oauthPlugin, options)
+    fastify.register(oauthPlugin, Object.assign({}, options, {
+      name: 'githubOAuth3',
+      startRedirectPath: '/login/github3'
+    }))
+
+    fastify.get('/', function (request, reply) {
+      return this.getAccessTokenFromAuthorizationCodeFlow(request)
+        .then(result => {
+          const token = this.githubOAuth2.accessToken.create(result)
```

Comment:
It seems that this api signature is not documented. Could you add also this example to readme?
This would explain better the "multi oauth2" feature